docker-compose.yml

```diff
@@ -123,6 +123,9 @@ services:
       - ${ST2_PACKS_DEV:-./packs.dev}:/opt/stackstorm/packs.dev:rw
       - stackstorm-virtualenvs:/opt/stackstorm/virtualenvs:rw
       - stackstorm-ssh:/home/stanley.ssh
+      # Action runner needs access to keys since action definitions (Jinja
+      # templates) can reference secrets
+      - stackstorm-keys:/etc/st2/keys:rw
```

Comment:
@armab Any reason that volume is mounted as writable? I don't think any of those services (client, api, actionrunner) need write access to it, just read one (only the container which generates it needs write access).

And in addition to that, I also don't think client needs or should have access to that volume (but we need to confirm).

Reply:
Yeah, read-only would work as well.

`st2client` needs access to everything (SSH keys, packs, configs, virtualenv,etc) for visibility/observation/ux reasons. Let users know what resources they have and what's possible to do with them as infra owner.

Reply:
ack.

To me it seems potentially risky since client container is something which should potentially also be available to non ops / admin users (so they shouldn't have access to the key itself), but as long that set up is not mean for prod deployments and we explicitly document that somewhere, it should be fine.

Reply:
So I guess for now, I'll change it to read only and add a note on why it's available to the client to the file + readme.

Reply:
st2client is just an interface for the users to easily work with deploment, including administration and full mgmt. Eg. it's already godmode and is granted/expected.

Readme includes the following scope of use for `st2-docker`:
> This docker-compose is provided as a way to allow "get up and running" quickly with StackStorm using Docker (based on st2-dockerfiles). It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.

Reply:
Yeah, in that case it should be fine (I might still add a comment on why it's there for st2client in the compose file :)).